README.md

```diff
@@ -6,11 +6,22 @@ The goal is to study the features required for such identification, when working
 the code of this project is arrange according to the architecture shown in https://github.com/asilev/NLP_FinalProject/blob/master/doc/architecture/NLP%20Author%20Classification%20Architecture.png.
 
 The files in TwitterReader are used to retrieve data sets from Twitter, using the twitter4j framework.
-This folder is meant to be compiled and run as a standalone executable with its own main method.
+The Twitter4j framework provides a pre-compiled JAR library, twitter4j-core-4.0.4.jar, which is imported into the project and then allows the access and extraction of data.
 Running it requires two parameters: The inpus path (this is unused, and can be anything) and the output path (where to put the tweets from tweeter).
+After initialization of the Twitter4j framework (accoridng to the Twitter API), the main API call we use is the "getUserTimeline" which accepts two parameters - user name and page ID.
```

Comment:
In my opinion, this is less important here. Maybe you can move it to the code documentation instead. Or you can leave it here, if you feel it helps the reader.